guides/doc-Provisioning_Guide/topics/Virt-Libvirt.adoc

```diff
@@ -161,16 +174,17 @@ To add KVM images on {ProjectServer}, complete the following steps:
 . From the *Architecture* list, select the operating system architecture.
 . In the *Username* field, enter the SSH user name for image access. This is normally the `root` user.
 . In the *Password* field, enter the SSH password for image access.
-. From the *User data* list, select if you want images to support user data input, such as `cloud-init` data.
 . In the *Image path* field, enter the full path that points to the image on the KVM server. For example:
 +
 [options="nowrap" subs="+quotes"]
 ----
- /var/lib/KVM/images/TestImage.qcow2
+ /var/lib/libvirt/images/TestImage.qcow2
```

Comment:
Note to change this downstream also. 
Are you sure about this @lzap? 
I had Kenny give me lots of feedback and test this very recently.

Reply:
It looks like somebody was replacing "libvirt" with "KVM" in this guide and this got replaced accidentaly. There is no such directory `/var/lib/KVM` and the default place for images is and has always been `/var/lib/libvirt/images`.